src/lib.rs

```diff
@@ -2668,29 +2722,23 @@ fn make_huffman_codes(lengths: &[usize]) -> Format {
     }
 
     let mut codes = Vec::with_capacity(lengths.len());
-    let mut branches = Vec::new();
+    let mut branches: Vec<Format> = Vec::new();
```

Comment:
`branches` does not appear to be used anymore. Is this intentional? (unused_mut, unused_variables).

Reply:
Unintentional, didn't notice the warning due to presence of the other warning :roll_eyes: